terraso_backend/apps/graphql/schema/shared_resources.py

```diff
@@ -51,3 +62,90 @@ def resolve_source(self, info, **kwargs):
 
     def resolve_target(self, info, **kwargs):
         return self.target
+
+    def resolve_download_url(self, info, **kwargs):
+        return f"{settings.API_ENDPOINT}/shared-data/download/{self.share_uuid}"
```

Comment:
```suggestion
        return f"{settings.API_ENDPOINT}/download/{self.share_uuid}"
```

Reply:
@paulschreiber This is the API endpoint, the urls are organized by app, so we can't avoid the `shared-data` without losing that structure